notebooks/Notebook1.md

```diff
@@ -2,33 +2,71 @@
+
+I also agreed that it is very helpful to have specific ideas prepared for class so I can get helpful feedback, so I considered how I will condense my main ideas here to highlight what I most need help with and also give a clear description of where I am so far.
 
 ## Description
 
-**TODO:** Fill in this part with information about your work this week:
-important design decisions, changes to previous decisions, open questions,
-exciting milestones, preliminary results, etc. Feel free to include images
-(e.g., a sketch of the design or a screenshot of a running program), links to
-code, and any other resources that you think will help clearly convey your
-design process.
+The main things I worked on this week were creating an [example output](<https://hmc-cs111-spring2023.github.io/project-code-occamkg/work%20files/basicPres0/example%20output>) and designing an [example syntax](<https://github.com/hmc-cs111-spring2023/project-code-occamkg/tree/master/work%20files/basicPres0/example%20syntax>).
+I spent most of my time on creating the output because I also needed to refamiliarize myself with web language programming.
+
+I got a lot of the basics down in terms of what I will need for every output.
+For example, in the JS file, I worked on how to translate the keyboard input into navigating the presentation timeline, which is a fundamental part of presentations.
+Similarly, in the style, I figured out how to make a content area that can be defined by the aspect ratio and fill the screen no matter its shape which is also essential for presentations that can function consistently.
+
+I kept the example pretty basic (only text elements so far), but I wanted to add capabilities that exemplify how these web languages can expand presentation capabilities, so I made sure to include transitions that affect style of different elements (I changed size and opacity of text).
+I also had fun messing around with transitions and tried to replicate typical slide transitions as simply as possible by creating a `flyOutRight` effect.
+
+For the syntax, I tried coming up with a way to express my example output efficiently and clearly (not very successfully).
+I separated out the class styling because I think that should just remain as pure CSS.
+I may change this if I want to add my own styles and have them be as easy to add to classes as the built-in styles.
 
 ## Questions
 
 **What is the most pressing issue for your project? What design decision do
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
 
+I must admit, the syntax I have is not great.
+I was fixated on the idea that I should have two sections: one for presentation elements and one for transitions, but it feels a bit unnatural.
+I was considering splitting them into separate files like HTML and JS, but since the transitions all use the elements by name, I'm not sure splitting the files would be best for the user.
```

Comment:
I was going back and forth with it a bit, but I think you should make it so that the code for the transitions and elements are in different files.